src/03_modules.py

```diff
@@ -10,22 +10,25 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+for x in sys.argv: 
```

Comment:
Never hurts to be more descriptive with your for loop variables! Ex `for arg in sys.argv`

Reply:
Perfect thank you